src/LastFiveValues.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const LastFiveValues = props => (
+  <p className="LastFiveSelectedValues" onClick={props.onClick}>
+    {props.value ? props.value : 'Click button'}
```

Comment:
Since value is initialized as `[]`, the "Click Button" branch never gets triggered.